docs/source/compilation/index.rst

```diff
@@ -0,0 +1,23 @@
+.. _compilation:
+
+Compilation
+=============
+
+Before starting Indico compilation, this guide assumes you've previously
+:ref:`cloned the development version <cloning>`.
```

Comment:
...and followed the dev setup guide

because if they *only* clone it they won't be able to build things

Reply:
Yes, I guess I can include the whole setup. I was trying to avoid pointing someone towards setting up mail-dump, the database and running.